frontend/class-frontend.php

```diff
@@ -915,7 +915,7 @@ private function generate_canonical() {
 				$canonical = get_search_link();
 			}
 			elseif ( is_front_page() ) {
-				$canonical = home_url();
+				$canonical = home_url( user_trailingslashit( '' ) );
```

Comment:
@moorscode `user_trailingslashit(home_url())` and `home_url(user_trailingslashit(''))` return the same results. The `ltrim` line I think you're referring to also adds a slash back in. The second version may be marginally faster as that whole line gets passed over if `$path` is empty.

I don't know that there is any other benefit for one way over the other so I don't have any strong preference.